packages/dropzone/README.md

```diff
@@ -107,3 +107,5 @@ When a file is dragged over the `<sp-dropzone>` element, it will display with th
 ## Accessibility
 
 When actions, e.g. copy/paste, can be enacted directly on the `<sp-dropzone>` element itself, be sure to supply a `tabindex` so that keyboard users can find this interaction in the tab order. For screen readers, supply appropriate `role` and `aria-label` attributes to clarify what these actions are and how to complete them.
+
+Also be sure that screen readers announce what was added to the dropzone. How this is achieved depends on what is displayed after the files are added. If the dropzone is replaced by the item added, direct focus onto that item by including the appropriate `role` and `aria-label` attributes.
```

Comment:
This is a little redundant with the above sentence `For screen readers, supply appropriate `role` and `aria-label` attributes to clarify what these actions are and how to complete them.`

Maybe we can remove the sentence on `109` and just keep the more in detail one on `111`